ChurchToolsApi.py

```diff
@@ -529,3 +529,67 @@ def get_event_agenda(self, event_id: int):
             return response_data
         else:
             logging.warning("Something went wrong fetiching events: {}".format(response.status_code))
+
+    def file_download(self, filename: str, domain_type, domain_identifier, path_for_download='./Downloads'):       
```

Comment:
@kolibri52 requires existance of Downloads folder
Any reason against using ./

Or did you include the folder as part of the commit?